eng/typescript-emitter-package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "@cadl-lang/compiler": "~0.38.5",
+    "@azure-tools/cadl-typescript": "1.0.0-beta.7",
```

Comment:
Lets try to limit this down to just the one package if possible.

Reply:
We cannot limit this down to just one package because the other dependencies are not listed in cadl-typescript or cadl-autorest. For `cadl-lang/compiler`, we also need to set it here because latest npm with command `npx cadl compile` will download latest cadl if it's not set.

Reply:
Can you help me understand what dependencies are missing from the cadl-typescript package that we need? It seems like if we want our strategy to work we need to figure that out. I though the single package works for the cadl-autorest and cadl-csharp emitters so I would expect it to work for this one too.